content/docs/setup/kubernetes/multicluster-install.md

```diff
@@ -42,35 +42,75 @@ Istio control plane, Envoy can then communicate with the **single**
 Istio control plane and form a mesh network across multiple Kubernetes
 clusters.
 
-## Deploy all Kubernetes clusters to be used in the mesh
-
-After deployment of remote clusters, each one will have a
-credentials file associated with the admin context typically
-located in `$HOME/.kube/config`.
+## Create service account in remote clusters and generate `kubeconfigs`
+
+The Istio control plane requires access to all clusters in the mesh to
+discover services, endpoints, and pod attributes.  The following
+describes how to create a Kubernetes service account in a remote cluster with
+the minimal RBAC access required.  The procedure then generates a `kubeconfig`
+file for the remote cluster using the credentials of the service account.
+
+The following procedure should be performed on each remote cluster to be
+added to the service mesh.  The procedure requires cluster-admin user access
+to the remote cluster.
+
+1. Create `ClusterRole`, `istio-reader`, for the Istio control plane access:
+
+     ```command
+      cat <<EOF | kubectl  create -f -
+      kind: ClusterRole
+      apiVersion: rbac.authorization.k8s.io/v1
+      metadata:
+        name: istio-reader
+      rules:
+         - apiGroups: ['']
+           resources: ['nodes', 'pods', 'services', 'endpoints']
+           verbs: ['get', 'watch', 'list']
+    EOF
+    ```
 
-## Gather credential files from remote
+1.  Create `ServiceAccount`, `istio-multi`, for the Istio control plane:
 
-> `${CLUSTER_NAME}` here is defined as the name of the remote
-cluster used and must be unique across the mesh.  Some Kubernetes
-installers do not set this value uniquely.  In this case, manual
-modification of the `${CLUSTER_NAME}` fields must be done.
+    ```command
+    $ export SERVICE_ACCOUNT=istio-multi
+    $ export NAMESPACE=istio-system
+    $ kubectl create ns ${NAMESPACE}
+    $ kubectl create sa ${SERVICE_ACCOUNT} -n ${NAMESPACE}
+    ```
 
-For each remote cluster, execute the following steps:
+1.  Bind `ServiceAccount`, `istio-multi`, to `ClusterRole` `istio-reader`:
```

Comment:
How about

Bind `ServiceAccount` named as `istio-multi` to `ClusterRole` named as  `istio-reader`:

Reply:
done